packages/core/src/tools/utils/polyfills.ts

```diff
@@ -50,33 +11,22 @@ export function findLast<T, S extends T>(
   return undefined
 }
 
-export function forEach<List extends { [index: number]: any }>(
-  list: List,
-  callback: (value: List[number], index: number, parent: List) => void
-) {
-  Array.prototype.forEach.call(list, callback as any)
-}
-
+// TODO remove after updating target to es2018
 export function objectValues<T = unknown>(object: { [key: string]: T }) {
   return Object.keys(object).map((key) => object[key])
 }
 
+// TODO remove after updating target to es2018
 export function objectEntries<T = unknown>(object: { [key: string]: T }): Array<[string, T]> {
   return Object.keys(object).map((key) => [key, object[key]])
 }
 
-export function startsWith(candidate: string, search: string) {
-  return candidate.slice(0, search.length) === search
-}
-
-export function endsWith(candidate: string, search: string) {
-  return candidate.slice(-search.length) === search
-}
-
 interface Assignable {
   [key: string]: any
 }
 
+// TODO: Object.assign generates larger bundle size than this polyfill.
+// try with spread operator after updating target to es2018
```

Comment:
**💭 thought:** ‏Yes, we should use the spread operator and remove [those ESLint rules](https://github.com/DataDog/browser-sdk/blob/3abdb5bd13651276dac3947b822d4fbd24022c33/.eslintrc.js#L237-L244) and enable [prefer-object-spread](https://eslint.org/docs/latest/rules/prefer-object-spread#rule-details) once fully migrated to ES2018. Exciting times 😄

Reply:
spoiler: https://github.com/DataDog/browser-sdk/pull/2908